experiments/integrated/performance/integrated_timestep_test.jl

```diff
@@ -265,19 +265,17 @@ h_leaf = FT(9.5) # m
 h_canopy = h_stem + h_leaf
 hydraulics = Canopy.PlantHydraulicsModel{FT}(
     canopy_domain,
-    LAI,
     toml_dict;
-    SAI,
-    RAI,
     n_stem,
     n_leaf,
     h_stem,
     h_leaf,
```

Comment:
In this case, it would make sense for h_leaf and h_steam to be included in biomass.

Reply:
I added h_canopy to the biomass component. The splitting of it into leaf vs stem is only required by the plant hydraulics model, so I left that as is.

(I added a consistency check so that if someone picks this plant hydraulics scheme, we ensure that h_canopy = h_leaf+h_stem.)